src/ql/resource/inter_core_channel.cc

```diff
@@ -185,26 +185,26 @@ void InterCoreChannelResource::on_initialize(rmgr::Direction direction) {
     }
 
     // Print result if debug is enabled.
-    QL_IF_LOG_DEBUG {
-        std::cout << "====================================" << std::endl;
-        std::cout << "resource instance " << context->instance_name;
-        std::cout << " of type " << context->type_name << std::endl;
-        std::cout << "------------------------------------" << std::endl;
-        std::cout << "incoming JSON:" << std::endl;
-        std::cout << context->configuration.dump(4) << std::endl;
-        std::cout << "------------------------------------" << std::endl;
-        std::cout << "desugared JSON:" << std::endl;
-        std::cout << config->json.dump(4) << std::endl;
-        std::cout << "------------------------------------" << std::endl;
-        std::cout << "preds[1q]: " << config->predicates[0] << std::endl;
-        std::cout << "preds[2q]: " << config->predicates[1] << std::endl;
-        std::cout << "preds[nq]: " << config->predicates[2] << std::endl;
-        std::cout << "inter_core_required: " << config->inter_core_required << std::endl;
-        std::cout << "communication_qubit_only: " << config->communication_qubit_only << std::endl;
-        std::cout << "num_cores: " << config->num_cores << std::endl;
-        std::cout << "num_channels: " << config->num_channels << std::endl;
-        std::cout << "====================================" << std::endl;
-    }
+//    QL_IF_LOG_DEBUG {
```

Comment:
see above

Reply:
Updated the pattern of QL_IF_LOG_DEBUG use and definition.